src/mbgl/map/map_context.cpp

```diff
@@ -92,10 +92,6 @@ void MapContext::triggerUpdate(const TransformState& state, const Update flags)
 }
 
 void MapContext::setStyleURL(const std::string& url) {
-    if (styleURL == url) {
```

Comment:
You're reverting 074d61a55e32e83173ec12b2eb34b78ddaaf0ef1 from earlier today. Accidental?